Makefile

```diff
@@ -15,7 +15,7 @@
 BINNAME     ?= move2kube-api
 BINDIR      := $(CURDIR)/bin
 DISTDIR		:= $(CURDIR)/_dist
-TARGETS     := darwin/amd64 linux/amd64
+TARGETS     := darwin/amd64 darwin/arm64 linux/amd64 linux/arm64 windows/amd64
```

Comment:
Aren't we supporting Darwin amd64? And windows arm64?

Reply:
uh it's the same as https://github.com/konveyor/move2kube/blob/main/Makefile#L19

Reply:
`windows/arm64` isn't supported by `gox` yet it seems https://github.com/mitchellh/gox/issues/154